tests/acceptance/test_dcos_package.py

```diff
@@ -24,8 +24,8 @@ def test_uninstall_package_and_wait():
     uninstall_package_and_wait('chronos')
     delete_zk_node('chronos')
     assert package_installed('chronos') == False
-    # this should only pass if we can install after the wait
-    install_package_and_wait('chronos')
+#    # this should only pass if we can install after the wait
+#    install_package_and_wait('chronos')
```

Comment:
issue is this:  https://github.com/dcos/shakedown/blob/dcoscli-051/shakedown/dcos/package.py#L28-L33
it is correct on master